staging/src/k8s.io/apiserver/pkg/util/flowcontrol/apf_controller.go

```diff
@@ -256,11 +292,15 @@ func (cfgCtlr *configController) Run(stopCh <-chan struct{}) error {
 	return nil
 }
 
+// this func is the logic of the one and only worker goroutine.  We
```

Comment:
nit: the golang conversion is to start comment with the name of the method/function

// runWorker is the logic of ...

Reply:
same in couple other places

Reply:
I did not realize this was expected in this sort of case, having been lulled into complacency by other sorts of cases that do not follow that convention.  I had assumed that the automated checking caught all the kinds of cases where following that convention is expected.